### DISCUSSION TITLE
I want talk with you

### DISCUSSION THREAD

**Discussion by wodaixin** [mmistakes/minimal-mistakes#3002] [General]
yesterday , i done the [chinese translate](https://docs.wodaixin.cn) with docs 4.22 .  but you release 4.23 , this's very interesting  :joy: :joy: :joy:  
![2021-06-15 22-56-39 的屏幕截图](https://user-images.githubusercontent.com/6753344/122075934-05bc2a80-ce2d-11eb-9ce6-a44afcdf230b.png)

**Comment by iBug** [comment]
😂
